libraries/AP_DDS/README.md

```diff
@@ -96,33 +96,14 @@ REBOOT
 
 ## Setup ROS 2 and micro-ROS
 
-Follow the steps to use the microROS Agent
-
-- Install ROS Humble (as described here)
-
-  - https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html
-
-- Install geographic_msgs
-  ```console
-  sudo apt install ros-humble-geographic-msgs
-  ```
-
-- Install and run the microROS agent (as described here). Make sure to use the `humble` branch.
-  - Follow [the instructions](https://micro.ros.org/docs/tutorials/core/first_application_linux/) for the following:
-
-    - Do "Installing ROS 2 and the micro-ROS build system"
-      - Skip the docker run command, build it locally instead
-    - Skip "Creating a new firmware workspace"
-    - Skip "Building the firmware"
-    - Do "Creating the micro-ROS agent"
-    - Source your ROS workspace
+Follow the [ArduPilot wiki here](https://ardupilot.org/dev/docs/ros2.html) to set up a ROS 2 workspace for communication with ArduPilot.
```

Comment:
Could consolidate this entire section into the "Installing Build Dependencies" section above which already includes a link to the wiki. Using "Follow the ArduPilot wiki [install ROS 2 and depenencies](...)" may be clearer.

Reply:
Good idea, done. I renamed the wording to make it no to tied to MicroXRCEDDSGen